deploy/addons/kubevirt/pod.yaml.tmpl

```diff
@@ -10,29 +10,27 @@ data:
   uninstall.sh: |
     #!/bin/bash
 
-    kubectl delete -f /manifests/kubevirt.yaml
+    kubectl delete -f /manifests/kubevirt-cr.yaml
 
-    kubectl delete -f /manifests/kubevirt-base.yaml
+    kubectl delete -f /manifests/kubevirt-operator.yaml
 
   install.sh: |
     #!/bin/bash
 
     export KUBEVIRT_VERSION=$(curl -s https://api.github.com/repos/kubevirt/kubevirt/releases | grep tag_name | grep -v -- - | sort -V | tail -1 | awk -F':' '{print $2}' | sed 's/,//' | xargs)
     echo $KUBEVIRT_VERSION
 
-    curl -Ls "https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-operator.yaml" -o /manifests/kubevirt-base.yaml
+    curl -Ls "https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-operator.yaml" -o /manifests/kubevirt-operator.yaml
+    kubectl create -f /manifests/kubevirt-operator.yaml
 
-    kubectl create -f /manifests/kubevirt-base.yaml
+    curl -sL "https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-cr.yaml" -o /manifests/kubevirt-cr.yaml
+    kubectl create -f /manifests/kubevirt-cr.yaml
 
     EMULATION=$(egrep 'svm|vmx' /proc/cpuinfo)
     if [ -z "$EMULATION" ]; then
       echo "use emulation"
-      kubectl create configmap kubevirt-config -n kubevirt --from-literal debug.useEmulation=true
-    fi;
-
-    curl -sL "https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-cr.yaml" -o /manifests/kubevirt.yaml
-
-    kubectl create -f /manifests/kubevirt.yaml
+      kubectl patch kubevirt kubevirt -n kubevirt --type merge --patch '{"spec": {"configuration": {"developerConfiguration": {"useEmulation": true}}}}'
```

Comment:
This definitely works, but in general our kubevirt-cr.yaml does not set any non-default option and for declarative kubevirt installs with kubevirt you can just `apply` such a `cr`:

```yaml
apiVersion: kubevirt.io/v1
kind: KubeVirt
metadata:
  name: kubevirt
  namespace: kubevirt
spec: {}
```

Or in your case:

```yaml
apiVersion: kubevirt.io/v1
kind: KubeVirt
metadata:
  name: kubevirt
  namespace: kubevirt
spec:
  developerConfiguration:
    useEmulation: true
```

So in stead of 

```
curl -sL "https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-cr.yaml" -o /manifests/kubevirt-cr.yaml
```

just directly write the above content.

Reply:
Updated accordingly.

Note that I too prefer the `cat << EOF | kubectl apply -f -` syntax, but this doesn't work in this context since the above `pod.yaml.tmpl` template is rendered using the `html/template` pkg, which doesn't like the `<<`.